mopidy_spotify/translator.py

```diff
@@ -183,7 +183,7 @@ def to_playlist(
         return models.Ref.playlist(uri=sp_playlist.link.uri, name=name)
     else:
         return models.Playlist(
-            uri=sp_playlist.link.uri, name=name, tracks=tracks
+            uri=sp_playlist.link.uri, name=name, tracks=list(tracks)
```

Comment:
This list call can be moved to together with `reversed()`. That should be the only branch that yields an iterator instead of a list.